config/locales/en.yml

```diff
@@ -933,9 +933,9 @@ help_text.requested_attributes.full_name: Full name
 help_text.requested_attributes.ial1_consent_reminder_html: You must consent each year to share your information with <strong>%{sp}</strong>. We’ll share your information with <strong>%{sp}</strong> to connect your account.
 help_text.requested_attributes.ial1_intro_html: We’ll share your information with <strong>%{sp}</strong> to connect your account.
 help_text.requested_attributes.ial2_consent_reminder_html: "<strong>%{sp}</stro\
-  ng> needs to know who you are to connect to your account. You must consent each year to share your verified information with <strong>%{sp}</strong>. We’ll share this information: "
-help_text.requested_attributes.ial2_intro_html: '<strong>%{sp}</strong> needs to know who you are to connect your account. We’ll share this information with %{sp}: '
-help_text.requested_attributes.ial2_reverified_consent_info: 'Because you verified your identity again, we need your permission to share this information with %{sp}: '
+  ng> needs to know who you are to connect to your account. You must consent each year to share your verified information with <strong>%{sp}</strong>. We’ll share this information:"
+help_text.requested_attributes.ial2_intro_html: '<strong>%{sp}</strong> needs to know who you are to connect your account. We’ll share this information with %{sp}:'
+help_text.requested_attributes.ial2_reverified_consent_info: 'Because you verified your identity again, we need your permission to share this information with %{sp}:'
```

Comment:
```
help_text.requested_attributes.ial2_consent_reminder_html
help_text.requested_attributes.ial2_intro_html
help_text.requested_attributes.ial2_reverified_consent_info
```

are all used here https://github.com/18F/identity-idp/blob/a79bba606b9a7225f7931f64c9fea3a5dfe4ccfe/app/presenters/completions_presenter.rb#L74 for https://github.com/18F/identity-idp/blob/a79bba606b9a7225f7931f64c9fea3a5dfe4ccfe/app/views/sign_up/completions/show.html.erb#L11 so the space removal should be safe.